### PR TITLE
fix perspective and app switcher to make them look more similar to ocp

### PR DESCRIPTION
**Pull request by willkutler**
Signed-off-by: Will Kutler <wkutler@redhat.com>

for https://github.com/open-cluster-management/backlog/issues/12265

### CODE REVIEW

File: src/AcmHeader/AcmHeader.tsx [hunk 1]
```diff
@@ -397,20 +403,34 @@ function NavExpandableList(props: { route: AcmRoute; showSwitcher: boolean; post
                         </DropdownToggle>
                     }
                     dropdownItems={[
+                        <DropdownItem onClick={() => setSwitcherOpen(false)} key={'acm'}>
+                            <Title headingLevel="h2" size="md">
+                                <span className="oc-nav-header__icon">
+                                    <svg viewBox="0 0 14 13.97" style={acmIconStyle}>
```
Comment (ChunxiAlexLuo) [inline src/AcmHeader/AcmHeader.tsx:409]: Can you subtract this? the rest look good to me

**Review by ChunxiAlexLuo — CHANGES_REQUESTED** [review]

**Review by ChunxiAlexLuo — APPROVED** [review]
/lgtm

**Comment by willkutler** [comment]
Need to refactor the ACM icon in the switcher into the ACMIcons folder, but going to put this in as is and refactor in a later PR since I'll be off the last few days of the sprint and would like to get this fix in first

**Comment by ChunxiAlexLuo** [comment]
approve first, need to subtract icon into acm icon folder later in another or

**Pull request merged**